ledgerjs-fio/test/integration/getPublicKey.test.ts

```diff
@@ -0,0 +1,87 @@
+import chai, {expect} from "chai"
+import chaiAsPromised from 'chai-as-promised'
+
+import type Fio from "../../src/fio"
+import {DeviceStatusError} from "../../src/fio"
+import {str_to_path} from "../../src/utils/address"
+import {getFio} from "../test_utils"
+import type {TestCase} from "./__fixtures__/getPublicKey"
+import {testsPublicKey} from "./__fixtures__/getPublicKey"
+
+chai.use(chaiAsPromised)
+
+const PublicKey = require('@fioprotocol/fiojs/dist/ecc/key_public')
+
```

Comment:
We can change this to import
```suggestion
import {Ecc} from '@fioprotocol/fiojs'
```
and then use it like this: `Ecc.PublicKey`